lib/circuitbox/circuit_breaker.rb

```diff
@@ -26,7 +26,9 @@ def initialize(service, options = {})
       @service = service
       @circuit_options = options
       @circuit_store   = options.fetch(:cache) { Circuitbox.circuit_store }
-      @notifier        = options.fetch(:notifier_class) { Notifier }
+
+      notifier_class = options.fetch(:notifier_class) { Notifier }
```

Comment:
I was always wondering if it wouldn't be easier to actually inject an initialized `notifier` same as is done for `circuit_store` maybe have `Notifier.default_notifier`. I'm always weary of injecting classes.